streams-tests/shared/src/test/scala/zio/stream/ZTransducerSpec.scala

```diff
@@ -18,7 +23,94 @@ object ZTransducerSpec extends ZIOBaseSpec {
     }
 
   def spec = suite("ZTransducerSpec")(
-    suite("Combinators")(),
+    suite("Combinators")(
+      suite("contramap")(
+        testM("happy path") {
```

Comment:
Can we cut down on some of these tests here and below? They're really just exercising `ZIO` at this point. Ideally we'd have property checks but not critical for now.